centreon/www/class/centreonDowntime.class.php

```diff
@@ -492,21 +476,17 @@ public function getForEnabledHostgroups()
                 h.host_name,
                 NULL as service_id,
                 NULL as service_description
-            FROM downtime_period dtp,
-                downtime dt,
-                downtime_hostgroup_relation dhr,
-                host h,
-                hostgroup_relation hgr,
-                hostgroup hg
-            WHERE dtp.dt_id = dhr.dt_id
-                AND dtp.dt_id = dt.dt_id
-                AND dhr.hg_hg_id = hgr.hostgroup_hg_id
-                AND hgr.host_host_id = h.host_id
-                AND hgr.hostgroup_hg_id = hg.hg_id
-                AND hg.hg_activate = '1';
-        SQL;
-
-        $statement = $this->db->query($request);
+            FROM downtime_period dtp
+            INNER JOIN downtime dt ON dtp.dt_id = dt.dt_id
+            INNER JOIN downtime_hostgroup_relation dhr ON dtp.dt_id = dhr.dt_id
+            INNER JOIN hostgroup_relation hgr ON dhr.hg_hg_id = hgr.hostgroup_hg_id
+            INNER JOIN host h ON hgr.host_host_id = h.host_id
+            INNER JOIN hostgroup hg ON hgr.hostgroup_hg_id = hg.hg_id
```

Comment:
To be more readable.
```suggestion
            INNER JOIN downtime dt
                ON dtp.dt_id = dt.dt_id
            INNER JOIN downtime_hostgroup_relation dhr
                ON dtp.dt_id = dhr.dt_id
            INNER JOIN hostgroup_relation hgr
                ON dhr.hg_hg_id = hgr.hostgroup_hg_id
            INNER JOIN host h
                ON hgr.host_host_id = h.host_id
            INNER JOIN hostgroup hg
                ON hgr.hostgroup_hg_id = hg.hg_id
```